runtime/vm/KeyHashTable.c

```diff
@@ -439,6 +441,28 @@ hashClassTableDelete(J9ClassLoader *classLoader, U_8 *className, UDATA className
 	return hashTableRemove(table, &key);
 }
 
+UDATA
+hashClassTablePackageDelete(J9VMThread *vmThread, J9ClassLoader* classLoader, J9ROMClass* romClass)
+{
+	if (isMHProxyPackage(romClass)) {
+		/* This generated package only exists for one hidden class and should be
```

Comment:
This still looks odd for me

Reply:
<img width="286" alt="image" src="https://github.com/eclipse-openj9/openj9/assets/12551021/32989ad5-5606-45e1-85fe-fbce1925e3f5">